ScipDotnet/ScipSymbolFormatter.cs

```diff
@@ -0,0 +1,376 @@
+    // balance between showing detailed/accurate information without using too verbose syntax.
+    private readonly SymbolDisplayFormat _format = new(
+        globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.OmittedAsContaining,
+        typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameOnly,
+        genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters |
+                         SymbolDisplayGenericsOptions.IncludeVariance |
+                         SymbolDisplayGenericsOptions.IncludeTypeConstraints,
+        memberOptions: SymbolDisplayMemberOptions.IncludeAccessibility |
+                       SymbolDisplayMemberOptions.IncludeModifiers |
+                       SymbolDisplayMemberOptions.IncludeParameters |
+                       SymbolDisplayMemberOptions.IncludeRef |
+                       SymbolDisplayMemberOptions.IncludeType |
+                       SymbolDisplayMemberOptions.IncludeConstantValue |
+                       SymbolDisplayMemberOptions.IncludeContainingType |
+                       SymbolDisplayMemberOptions.IncludeExplicitInterface,
+        delegateStyle: SymbolDisplayDelegateStyle.NameAndSignature,
+        extensionMethodStyle: SymbolDisplayExtensionMethodStyle.InstanceMethod,
+        parameterOptions: SymbolDisplayParameterOptions.IncludeType |
+                          SymbolDisplayParameterOptions.IncludeName |
+                          SymbolDisplayParameterOptions.IncludeDefaultValue |
+                          SymbolDisplayParameterOptions.IncludeExtensionThis |
+                          SymbolDisplayParameterOptions.IncludeOptionalBrackets |
+                          SymbolDisplayParameterOptions.IncludeParamsRefOut,
+        propertyStyle: SymbolDisplayPropertyStyle.ShowReadWriteDescriptor,
+        localOptions: SymbolDisplayLocalOptions.IncludeType |
+                      SymbolDisplayLocalOptions.IncludeRef |
+                      SymbolDisplayLocalOptions.IncludeConstantValue,
+        kindOptions: SymbolDisplayKindOptions.IncludeTypeKeyword |
+                     SymbolDisplayKindOptions.IncludeMemberKeyword |
+                     SymbolDisplayKindOptions.IncludeNamespaceKeyword,
+        miscellaneousOptions: SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral |
+                              SymbolDisplayMiscellaneousOptions.UseSpecialTypes |
+                              SymbolDisplayMiscellaneousOptions.UseAsterisksInMultiDimensionalArrays |
+                              SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier |
+                              SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers
+    );
+
+    public ScipSymbolFormatter(
+        Document doc,
+        IndexCommandOptions options,
+        Dictionary<ISymbol, ScipSymbol> globals)
+    {
+        _doc = doc;
+        _options = options;
+        _globals = globals;
+        _languagePrefix = _doc.Language == "C#" ? "cs" : "vb";
```

Comment:
nit: I had to check how this variable was used to understand what it does, this name might be clearer
```suggestion
        _markdownCodeFenceLanguage = _doc.Language == "C#" ? "cs" : "vb";
```